pkg/controller/provider/container/ovirt/resource.go

```diff
@@ -13,10 +13,11 @@ type System struct {
 		Name    string `json:"name"`
 		Vendor  string `json:"vendor"`
 		Version struct {
-			Build    string `json:"build"`
-			Major    string `json:"major"`
-			Minor    string `json:"minor"`
-			Revision string `json:"revision"`
+			Build       string `json:"build"`
+			Major       string `json:"major"`
+			Minor       string `json:"minor"`
+			Revision    string `json:"revision"`
+			FullVersion string `json:"full_version"`
```

Comment:
do we use `FullVersion` anywhere?

Reply:
yes, in the version method above. unfortunately revision is usually set to `0` even so the full version differ.

Reply:
ah ok, that's the struct we read from - so do we need the other properties of the Version struct? it looks like we have too much stuff here

Reply:
not really. but it doesn't harm. we query ovirt API anyway, it gets these details. i don't see a reason to drop them. although we can.

Reply:
because only full version is the way to determine the exact ovirt version.
e.g:
```
<version>
  <build>3</build>
  <full_version>4.5.3.9-1.el8ev</full_version>
  <major>4</major>
  <minor>5</minor>
  <revision>0</revision>
</version> 
```

i'm ok overall to drop it. since there is no real use of these values. but we query them anyway.